src/test/java/io/vertx/core/net/impl/pool/StressTest.java

```diff
@@ -218,6 +240,14 @@ protected void onSuccess(FakeConnection conn) {
       latch.countDown();
     });
     awaitLatch(latch);
+    if (resumeExecutor != null) {
+      // maybe there's still a suspended continuation waiting to be resumed:
+      // given that's a single threaded executor we just need to enqueue past to it
+      // and no new continuations should be in-flights, because after closing the pool
+      // there won't be any submitted actions
+      resumeExecutor.submit(() -> {}).get();
```

Comment:
that's a cheap trick IMO @vietj 
My concern is that even in the real use case, if the current continuation is still suspended ie yield happened, ready to be executed on some event loop
none knows about it! Nor the pool or the combiner too